File: Patient/Patient/DTOs/ClinicDto.cs
namespace Patient.DTOs;

public class ClinicDto
{
    public string id { get; set; }
    public string name { get; set; }
    public string address { get; set; }
    
    public ClinicDto (string id, string name, string address)
    {
        this.id = id;
        this.name = name;
        this.address = address;
    }

}

File: PatientsServer/src/main/java/DTOs/ClinicDto.java
package DTOs;

public class ClinicDto
{
  private String id;
  private String name;
  private String address;

  public ClinicDto(String id, String name, String address)
  {
    this.id = id;
    this.name = name;
    this.address = address;
  }

  public String getId()
  {
    return id;
  }

  public String getAddress()
  {
    return address;
  }

  public String getName()
  {
    return name;
  }

  public void setId(String id)
  {
    this.id = id;
  }

  public void setName(String name)
  {
    this.name = name;
  }

  public void setAddress(String address)
  {
    this.address = address;
  }
}
